circle_suck/base/constants.py

```diff
@@ -1,5 +1,16 @@
 from base.utils import merge_dicts
 
+# list of sports as choices.
+SPORTS_CHOICES = [
+    ('Football', 'Football')
+]
+
+# list of conference ID's to conference name pairs
+
+CONF_CHOICES = [
+    ('PAC_12','Pac-12'),
```

Comment:
these values are already in `CONFERENCES`, so do `[(conference_id, conference['name']) for conference_id, conference in CONFERENCES.items()]`